requirements/requirements.win.txt

```
@@ -11,7 +11,7 @@ cffi==1.15.1
 colorama==0.4.6
 comm==0.1.3
 contourpy==1.0.7
-cryptography==41.0.1
+cryptography==41.0.0
```

41.0.1 > 41.0.0, both are safe, no changing back
```suggestion
cryptography==41.0.1
```